app/main/views/questionnaire.py

```diff
@@ -70,39 +78,41 @@ def get_summary(eq_id, form_type, collection_id):
     navigator = g.questionnaire_manager.navigator
     latest_location = navigator.get_latest_location(get_answers(current_user), get_completed_blocks(current_user))
     if latest_location is 'summary':
-        return render_page('summary', True)
+        metadata = get_metadata(current_user)
+        answers = get_answers(current_user)
+        schema_json = _render_schema(g.schema_json, answers, metadata)
```

Comment:
These lines are basically the same as in _questionnaire_rendering_context()
Could we combine these somehow?